src/script/ceph-release-notes

```diff
@@ -96,9 +110,14 @@ if __name__ == "__main__":
                         help="path to ceph git repo")
     parser.add_argument("--token", default=os.getenv("GITHUB_ACCESS_TOKEN"),
                         help="Github Access Token ($GITHUB_ACCESS_TOKEN otherwise)")
+    parser.add_argument("--original_issue", dest='original_issue', action='store_true',
```

Comment:
oops I changed to --original-issue as the default now, should I change it back ?